README.md

```diff
@@ -43,7 +43,39 @@ For local development, you can copy `.env.example`
 cp .env.example .env
 ```
 
-### 3. Start the application
+### 3. Enable pre-commit hooks
+
+This project uses **pre-commit** to run Ruff linting and formatting before each commit.
+
+> ℹ️ The following commands only need to be run once per developer.
+
+Install development dependencies (including `pre-commit` and `ruff`):
+
+```bash
+uv sync --group dev
+```
+
+Enable the Git hook:
+
+```bash
+uv run pre-commit install
+```
+
+(Optional) Run all checks manually:
+
+```bash
+uv run pre-commit run --all-files
+```
+
+> 💡 Alternative (pip instead of uv):
+>
+> ```bash
+> pip install pre-commit      # Install
+> pre-commit install          # Enable 
+> pre-commit run --all-files  # Run checks
```

Comment:
`pre-commit uninstall #Disable` to be added when we add the above suggested section.